freqtrade/plugins/pairlist/pairlist_helpers.py

```diff
@@ -40,3 +40,15 @@ def expand_pairlist(wildcardpl: List[str], available_pairs: List[str],
             except re.error as err:
                 raise ValueError(f"Wildcard error in {pair_wc}, {err}")
     return result
+
+
+def dynamic_expand_pairlist(config: dict, markets: list) -> List[str]:
+    if config.get('freqai', {}):
+        corr_pairlist = config['freqai']['feature_parameters']['include_corr_pairlist']
+        full_pairs = config['pairs'] + [pair for pair in corr_pairlist
+                                        if pair not in config['pairs']]
+        expanded_pairs = expand_pairlist(full_pairs, markets)
```

Comment:
I think you will want to add `corr_pairlist` pairs **after** the `expand_pairlist()` call.

Assuming i use `include_corr_pairlist` of `.*/USDT` - which means all USDT pairs available on the exchange (now that's excessive as it'll be a few 100 pairs on binance - but other wildcard pairs can cause similar scenarios).

This sounds good in practice - but is a moving target, depending on pair listings / delistings.
In my understanding, this would then break freqai, as the number of features would be changing over time - depending on listings / delistings of the exchange